### PR TITLE
Fixes a typo in species.clear_organs which would cause it to not do its damn job

### PR DESCRIPTION
**Pull request by gbasood**

### CODE REVIEW

**Comment by PJB3005** [comment]
Wait so you're saying there's 2 procs with the same name barring capitalisation? 

That's worth refactoring in and of itself.

**Comment by gbasood** [comment]
Actually now that I think about I'm not 100% sure on what the desired behavior is here.
I'll give look at where clear_organs is used later

**Comment by gbasood** [comment]
``` DM
/datum/organ/proc/Remove(var/mob/living/carbon/human/H, var/mob/surgeon=null)
    return 1
```

That's the base organ removal proc

``` DM
/datum/organ/internal/proc/remove(var/mob/user, var/quiet=0)


    if(!removed_type) return 0

    var/obj/item/organ/removed_organ = new removed_type(get_turf(user))

    if(istype(removed_organ))
        removed_organ.organ_data = src
        removed_organ.update()
        organ_holder = removed_organ

    return removed_organ
```

And that's the internal organ removal proc.

What the fuck is this code

**Comment by Kitfox88** [comment]
shit

the word you're looking for is shit

**Comment by gbasood** [comment]
Well I don't know how to refactor those properly
I'm gonna just fix the proc call and hope it turns out to be desired behavior

**Comment by gbasood** [comment]
Yeah okay this just seems like it was a typo on N3X's part
Good to go

File: code/modules/mob/living/carbon/species.dm [hunk 1]
```diff
@@ -142,7 +142,9 @@ var/global/list/whitelisted_species = list("Human")
 		H.organs.len=0
 	if(H.internal_organs)
 		for(var/datum/organ/internal/I in H.internal_organs)
-			I.Remove(H, quiet=1) // GET OUT REEEEEEE
+			// I.Remove(H) // THIS DOES NOTHING AT THE MOMENT
+			I.owner = null
```
Comment (clusterfack) [inline code/modules/mob/living/carbon/species.dm]: Just add this to Destroy desu

**Pull request merged**